LICENSE

```diff
@@ -1,6 +1,8 @@
-MIT License
+The MIT License (MIT)
```

Comment:
Has the heading of the license changed?

GitHub's autogenerated one just says MIT License: https://choosealicense.com/licenses/mit/

Reply:
```suggestion
MIT License
```